unittests/test_launchers.py

```diff
@@ -126,10 +126,10 @@ def expected_command(self):
     @property
     def expected_minimal_command(self):
         return ('srun '
-                '--job-name=fake_job '
+                '--job-name=rfm_fake_job '
                 '--time=0:10:0 '
-                '--output=fake_job.out '
-                '--error=fake_job.err '
+                '--output=rfm_fake_job.out '
+                '--error=rfm_fake_job.err '
```

Comment:
Only the job name should be prefixed.

Reply:
Basically, you have to change the unit tests and pass explicitly the `stdout` and `stderr` arguments to the `FakeJob`. We need to test the possibility for these being different, because this is how it is called from the `RegressionTest`.

Reply:
I think that passing explicitly `stdout` and `stderr` is what `self.job` does. In case of the `self.minimal_job` only the required arguments are given and thus the `stdout` and `stderr` are named according to the job name.

Reply:
They should be passed as well to emulate the desired behaviour. We are not expecting the stdout/stderr to be prefixed.

Reply:
Then the passed name and not the actual `job` field should be passed to the `stdout/stderr` if they are `None` in the setup.